Ports/qt6-qtbase/package.sh

```diff
@@ -12,16 +12,18 @@ configopts=(
     "-DQT_BUILD_TOOLS_WHEN_CROSSCOMPILING=ON"
     "-DQT_HOST_PATH=/usr"
     "-DQT_FEATURE_cxx20=ON"
+    "-DQT_FEATURE_ssl=ON"
+    "-DQT_FEATURE_zstd=ON"
```

Comment:
Is there a QT feature for disabling IPv6? We've got a ton of IPv6 declarations and such to get ports to compile, but as of yet none of it actually works 😅

Reply:
Sadly, I didn't find any features for doing something like that...